reinforcement_learning/rlclientlib/Makefile

```diff
@@ -5,9 +5,12 @@ INCLUDE = -I ../include -I ../../explore -I ../explore -I ../../rapidjson/includ
 
 .PHONY: default all clean
 
-default: $(TARGET)
+default: pre-build $(TARGET)
 all: default
 
+pre-build:
```

Comment:
The makefile rule should specify the output of this command as the target. This target should then be a dependency for every target that relies on it

Reply:
Not sure what does this mean. Let's take it offline.